src/notebooks/R Mia Examples/_resources/mgnifyr_help.md

```diff
@@ -0,0 +1,20 @@
+# Help with MGnifyR
+
+MGnifyR is an R package that provides a convenient way for R users to access data from [the MGnify API](https://www.ebi.ac.uk/metagenomics/api/).
+
+Detailed help for each function is available in R using the standard `?function_name` command (i.e. typing `?mgnify_query` will bring up built-in help for the mgnify_query command). 
+
+A vignette is available containing a reasonably verbose overview of the main functionality. 
+This can be read either within R with the `vignette("MGnifyR")` command, or [in the development repository](https://htmlpreview.github.io/?https://github.com/beadyallen/MGnifyR/blob/master/doc/MGnifyR.html)
+
+## MGnifyR Command cheat sheet
+
+The following list of key functions should give a starting point for finding relevent documentation.
```

Comment:
Could we here instead refer to the MGnifyR website? e.g. function reference
https://ebi-metagenomics.github.io/MGnifyR/reference/index.html

-> this could reduce maintenance burden if there are updates in function names etc.